tests/unit/models/blueprint-test.js

```diff
@@ -1046,6 +1046,38 @@ describe('Blueprint', function() {
     var blueprint;
     var ui;
     var tmpdir;
+
+    beforeEach(function() {
+      tmpdir    = tmp.in(tmproot);
+      blueprint = new Blueprint(basicBlueprint);
+      ui        = new MockUI();
+    });
+
+    afterEach(function() {
+      return remove(tmproot);
+    });
+
+    it('passes a packages array for addAddonsToProject', function() {
+      blueprint.addAddonsToProject = function(options) {
+        expect(options.packages).to.deep.equal(['foo-bar']);
+      };
+
+      blueprint.addAddonToProject('foo-bar');
+    });
+
+    it('passes a packages array with target for addAddonsToProject', function() {
+      blueprint.addAddonsToProject = function(options) {
+        expect(options.packages).to.deep.equal([{name: 'foo-bar', target: '^123.1.12'}]);
+      };
+
+      blueprint.addAddonToProject({name: 'foo-bar', target: '^123.1.12'});
+    });
```

Comment:
@mike-north I'm not sure what that would accomplish.  I could literally pass *anything* in target and the result would be the same.

For example: 
```
it('passes a packages array with target for addAddonsToProject', function() {
      blueprint.addAddonsToProject = function(options) {
        expect(options.packages).to.deep.equal([{name: 'foo-bar', target: '^123.1.12'}]);
      };

      blueprint.addAddonToProject({name: 'foo-bar', target: 'banana pudding pops'});
    });
```

addAddonToProject/addAddonsToProject aren't responsible for sanitizing the inputs, any more than addPackage(s)ToProject. This is something that is (or should be) handled in a more centralized place downstream, probably in `addon-install` or add`npm-task`.